src/apps/loyalty/server/index.tsx

```diff
@@ -69,6 +70,18 @@ app.get("/inquiries", (req, res) => {
     return res.redirect(req.baseUrl + "/login")
   }
 
+  const headers = { "X-Access-Token": req.user.get("accessToken") }
```

Comment:
Can you move that to the CurrentUser model which is already fetching the user from gravity?

https://github.com/artsy/reaction-force/blob/master/src/apps/loyalty/server/current_user.ts